targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2020-2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-include(${MBED_PATH}/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/mbed_set_post_build_tfm.cmake)
+include(mbed_set_post_build_tfm)
```

Comment:
this fails to build as this target might be processed prior TFM (where the scripts are) and include fails to find it.
The problem is we moved scripts into libraries and libraries can be loaded later. `mbed_set_post_build` - this works as the module path is updated prior adding any additional Mbed OS library.

There are dependencies between targets and libraries like tfm and others .

Reply:
Deps like this should be resolved as we are loading first generic modules, target as the last. Checking why it can't find it.

Reply:
No, it bombs very early so CMAKE_MODULE_PATH must be configured early in the stage.

The error I get now:
```
CMake Error at mbed-os/targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/CMakeLists.txt:4 (include):
  include could not find load file:

    mbed_set_post_build_tfm
```